### DISCUSSION TITLE
Patch name should be seen from the bank page

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#683] [Ideas]
When you access the bank page of a TG you have this screen : 

Bank-----------TG X
<XXXX=Name of the bank>

Then if you press a MIDI Button associated to PgmUp/Down, you can switch from one voice to another but the screen don't reflect anything. In this case, first line should be replaced by patch name

**Comment by probonopd** [comment]
Agree this would be nice to have. And the handling of voices vs. performances should be made more consistent.
